src/python_bindings/py_algorithm.h

```diff
@@ -47,6 +87,77 @@ class PyAlgorithmBase {
     void LoadData();
 
     pybind11::int_ Execute(pybind11::kwargs const& kwargs);
+    std::unordered_map<std::string, std::string> GetOpts() {
+        std::unordered_map<std::string_view, boost::any> opt_ = algorithm_->GetOptValues();
+        std::unordered_map<std::string, std::string> res;
+        // for (auto& i:converters){
+        //     std::cout<<i.first<<" "<<'\n';
+        // }
+        for (auto& i : opt_) {
```

Comment:
```suggestion
        for (auto& [name, value] : opt) {
```
Why not `auto const&`?